packages/react-components/react-avatar/config/api-extractor.local.json

```diff
@@ -1,5 +1,5 @@
 {
   "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
   "extends": "./api-extractor.json",
-  "mainEntryPointFilePath": "<projectFolder>/dist/packages/react-components/<unscopedPackageName>/src/index.d.ts"
+  "mainEntryPointFilePath": "<projectFolder>/dist/types/packages/react-components/<unscopedPackageName>/src/index.d.ts"
```

Comment:
Out of curiosity, why is this change needed in this PR?

Reply:
`create-component` added it, it doesn't seem to affect anything though. I can remove it just in case?

Reply:
I'm not really sure what this actually does. But if it was added by create-component, and it doesn't break things, then it should be fine.